src/modules/service/components/Service.tsx

```diff
@@ -13,31 +13,41 @@ import useInterfaces from "shared/hooks/useInterfaces";
 import useDownloadServiceStream from "../../../shared/hooks/useDownloadServiceStream";
 import useGetServices from "../../../shared/hooks/useGetServices";
 import useSettingStore from "../../../shared/store/setting";
+import type { App } from "../types";
 
 import CustomServiceCard from "./CustomServiceCard";
 import SearchFilter from "./SearchFilter";
 import ServiceCard from "./ServiceCard";
 
-// TODO: appId is not a url param anymore
 const Service = () => {
-  const { appId } = useParams();
-
   const {
     data: services,
     isLoading: isServicesLoading,
     refetch: refetchServices,
   } = useGetServices();
   const { data: apps } = useInterfaces();
+  const [appsAugmented, setAppsAugmented] = useState<App[]>([]);
   const progresses = useSettingStore((state) => state.serviceDownloadsInProgress);
   const { mutate: download } = useDownloadServiceStream();
   const downloadingServices = useSettingStore.getState().downloadingServices;
   const [filter, setFilter] = useState(new Map<string, boolean>());
 
+  useEffect(() => {
+    const _apps = apps?.concat({
+      id: "available",
+      name: "Available",
```

Comment:
```suggestion
      name: "Only Available on this device",
```
Would it be too long?

Reply:
Yes it would be too long for a tag.